docs/src/auth.md

```diff
@@ -260,58 +268,64 @@ test('test', async ({ page }) => {
 ```js tab=js-js
 const { test } = require('@playwright/test');
 
+// Name of the storage state entry. The entry is saved in the project setup.
+test.use({
+  storageStateName: 'outlook-test-user'
+})
+
 test('test', async ({ page }) => {
   // page is signed in.
 });
 ```
 
-:::note
-If you can log in once and commit the `storageState.json` into the repository, you won't need the global setup at all, just specify the `storageState.json` in Playwright Config as above and it'll be picked up.
+### Committing storage state int the repository
+* langs: js
 
-However, periodically, you may need to update the `storageState.json` file if your app requires you to re-authenticate after some amount of time. For example, if your app prompts you to sign in every week even if you're on the same computer/browser, you'll need to update `storageState.json` at least this often.
-:::
+When you set an entry on [`method: TestInfo.storage`] Playwright will store it in a separate file under `.playwright-storage/`. Playwright does not delete those files automatically. You can leverage this fact to persist storage state between test runs and only sign in if the entry is not in the storage yet.
+
+If you can log in once and commit the storage state into the repository, you won't need the setup step test that signs in at all, just specify the name of the storage entry as above and it'll be picked up.
+
+However, you may need to periodically update the storage state entry if your app requires you to re-authenticate after some amount of time. For example, if your app prompts you to sign in every week even if you're on the same computer/browser, you'll need to update saved storage state at least this often.
```

Comment:
I'd swap this and previous paragraph, and recommend removing `./playwright-storage` directory for an easy referesher.